packages/react-core/src/components/Button/Button.tsx

```diff
@@ -171,7 +171,8 @@ const ButtonBase: React.FunctionComponent<ButtonProps> = ({
       {isLoading && (
         <span className={css(styles.buttonProgress)}>
           <Spinner
-            size={spinnerSize.md}
+            size={!isInline && spinnerSize.md}
```

Comment:
Is the `!isInline` check unnecessary considering line 47 in Spinner.tsx?  `isInline ? styles.modifiers.inline : styles.modifiers[size]`

Reply:
You should  just need to pass `isInline` to the spinner here and not have this check.

Reply:
nice, thanks!